### ISSUE TITLE
Too many logs at /tmp, can't disable them

### ISSUE THREAD

**Issue by teresaejunior** [MisterTea/EternalTCP#63]
Hello! I have noticed there are tens of */tmp/etserver** files in the server, and tens of */tmp/etclient** in the client. I tried running `et -l=/dev/null`, but it makes no difference.

Since `et` has been running mostly well, I would like to disable logs on both the server and client!

**Comment by MisterTea** [comment]
Hey, FWIW you should only get one of these logs per et connection, which should happen very infrequently since the connection is supposed to last for weeks/months.

With that said, I agree that we should have a way of turning them off (a config param for etserver and a -s switch for etclient).

**Comment by teresaejunior** [comment]
Yes, it's not that much of a problem, but it happens to me because I close the session whenever I finish what I was supposed to do.

**Comment by teresaejunior** [comment]
Thank you!
